cmake/ginkgo.pc.in

```diff
@@ -1,10 +1,10 @@
-prefix=@PKG_CMAKE_INSTALL_PREFIX@
-libdir=@CMAKE_INSTALL_FULL_LIBDIR@
-includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+prefix=@GINKGO_PKG_INSTALL_PREFIX@
+libdir=@GINKGO_INSTALL_FULL_LIBDIR@
+includedir=@GINKGO_INSTALL_FULL_INCLUDEDIR@
```

Comment:
I think ideally the libdir and includedir should be relative to the install prefix

Reply:
My main thought is that this makes it harder for packagers who want to patch pkg-config files with a different location, only a single line to change instead of 3

Reply:
Thanks, I see. That should be an easy fix.